examples/tutorial/hello/go/README.md

```diff
@@ -1,66 +1,185 @@
-# Tutorial 1: Go Function w/ Input (3 minutes)
+# Java Function Hello World
```

Comment:
This is in the Go directory, but has a bunch of Java stuff and Go stuff mixed together.

Reply:
Let me look.

Reply:
Yep. Fixed it. Also noticed a few other formatting issues. This are all fixed now as well.